pynecone/.templates/web/utils/state.js

```diff
@@ -70,6 +70,9 @@ export const applyDelta = (state, delta) => {
  */
 export const applyEvent = async (event, router, socket) => {
   // Handle special events
+
+  event.router_data = router;
```

Comment:
There seems to be many extra fields in `router` - we can subselect to just the ones we need:
```
event.router_data = (({ pathname, query }) => ({ pathname, query }))(router);
```